policy/exposures.yaml

```diff
@@ -15,12 +15,14 @@ deepDiff_differenceExp:
     python:        lsst.afw.image.ExposureF
     persistable:   ExposureF
     storage:       FitsStorage
+    level:         Ccd
```

Comment:
In the DecamMapper, every instance of setting level sets it to an explicit string "ccd" (lowercase and with quotes). Does this matter? If not, then it looks good to me!